_pages/dev/ssr/server-side-rendering-contextual-logging.md

```diff
@@ -252,3 +252,115 @@ You can also customize the behavior of the `LoggerService` by providing your own
 ### LoggerService Behavior in an SSR Context
 
 For server-side rendering, Spartacus overwrites the `LoggerService` with the `ExpressLoggerService` by default, which uses the `DefaultExpressServerLogger` for logging under the hood.
+
+## Trace Context
+
+In addition to standardized SSR logging, Spartacus offers support for the [W3C Trace Context](https://www.w3.org/TR/trace-context/) specification. Using the build-in parser in Spartacus, you can extract trace context information from incoming requests and include it in the logs generated during server-side rendering. This enhancement provides new opportunities for seamlessly linking logs with the traces of requests. It is now possible to link the logs displayed in one tool to the distributed trace presentation in another tool (for example, linking logs displayed in OpenSearch to the distributed trace presentation in Dynatrace).
+
+**Note:** Using the trace context standard has the potential to introduce security risks. To mitigate the risk of buffer overflow and HTML injection attacks, vendors using the `traceparent` header must follow best practices for parsing headers that could be malicious. It's important to be aware that `traceparent` may expose sensitive information, which could potentially facilitate more significant attacks. Precautions should be taken accordingly. Enabling distributed tracing without adequate safeguards can leave systems vulnerable to denial-of-service attacks, so tracing vendors should implement protective measures, such as handling authenticated and unauthenticated requests differently, and implementing rate limiters. While these particular risks are not specific to Spartacus itself, if you have concerns regarding the security of the `traceparent` header, you may want to consider creating your own `ExpressServerLogger` implementation that does not include `traceContext` values in the logs.
+
```

Comment:
When it mentions vendors in this note, I am wondering who we are talking about... Vendors such as Dynatrace and OpenSearch? Or do we mean Spartacus customers who are setting up trace context for their storefront?

Reply:
I was referring to observability platforms such as Dynatrace (there are other solutions on the market as well) that attach the `traceparent` to requests for distributed tracing purposes.

Reply:
That's what I thought, and then... I wasn't sure 😄 So thank you for confirming! 😊